fe/fe-core/src/main/java/com/starrocks/transaction/TransactionState.java

```diff
@@ -398,12 +399,14 @@ public void setTabletCommitInfos(List<TabletCommitInfo> infos) {
 
     public boolean tabletCommitInfosContainsReplica(long tabletId, long backendId) {
         TabletCommitInfo info = new TabletCommitInfo(tabletId, backendId);
-        if (this.tabletCommitInfos == null || this.tabletCommitInfos.contains(info)) {
+        if (this.tabletCommitInfos == null) {
+            Backend backend = GlobalStateMgr.getCurrentSystemInfo().getBackend(backendId);
             // if tabletCommitInfos is null, skip this check and return true
+            LOG.warn("tabletCommitInfos is null in TransactionState, tabletid {} backend {} transid {}",
+                    tabletId, backend != null ? backend.toString() : "", transactionId);
```

Comment:
Please try to keep the names and abbreviations in the log as consistent as possible!